rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -288,11 +291,25 @@ cd %{_sourcedir}
 %define no_ldso true
 %endif
 
+#### Use systemd everywhere except on: AIX, RHEL<7, SLES<12
```

Comment:
Don't we have any other old distro like fedora ?

Reply:
I will add the Fedora<15 condition.